### DISCUSSION TITLE
Is there a nix flake for tendermint core?

### DISCUSSION THREAD

**Discussion by nathan-at-least** [informalsystems/cosmos.nix#139] [Q&A]
I'm experimenting with writing an abci app as an external process to `tendermint` core and interested in using `nix`. I'm not too familiar with nix and poked around the source here to see if `tendermint` binary appears as a nixified dependency.

I couldn't find it anywhere, although I see `mkCosmosGoApp` accepts a "tendermint/tendermint" parameter. Just based on the name I wondered if this bundles a go appchain into tendermint within a monolithic process, whereas I seek to run an external process (written in rust).

Does this repository support my use case? If not, what would it take to add that support?

**Comment by romac** [comment]
> Just based on the name I wondered if this bundles a go appchain into tendermint within a monolithic process

Yes that's exactly it.

As you've figured out, we don't provide a Tendermint/CometBFT package in the repo right now, for no reason other than we haven't needed it yet for our own use, but would happily accept a PR to add CometBFT :)
